tests/conformance_tests/tests/conformance_runner.rs

```diff
@@ -289,9 +295,15 @@ async fn conformance_test_runner() {
         .await
         .unwrap();
 
-    let walker = WalkDir::new("test-vectors/corpus")
-        .into_iter()
-        .chain(WalkDir::new("extra-vectors").into_iter());
+    let walker = match interpreter::Backend::get_backend_choice() {
+        interpreter::Backend::FVM => WalkDir::new("fvm-test-vectors/corpus").into_iter(),
+        interpreter::Backend::Native | interpreter::Backend::Both => {
```

Comment:
Don't we want to run the FVM test vectors in case of `Backend::Both`?

Reply:
Hmm, in an ideal world, I think we would always just use the FVM test vectors. They're the only ones guaranteed to be up-to-date.

Reply:
The conformance tests are now using the new repository of test-vectors. There are a few hundred more than in the previous repository and they work with both the backends.